app/views/shared/_nav.html.erb

```diff
@@ -0,0 +1,23 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light px-5">
+  <a class="navbar-brand" href="#">My Recipes</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item mx-3">
+        <%= link_to "Public", root_path, class: "nav-link #{'active' if current_page?(root_path)}" %>
```

Comment:
- [ ] Kindly modify the link to display "Public recipes", this will help to be more descriptive for the user